src/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -49,12 +49,52 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Collect metrics about the alignment of RNA to various functional classes of loci in the genome:" +
-                "coding, intronic, UTR, intergenic, ribosomal. Also determines strand-specificity for strand-specific libraries.",
-        usageShort = "Produces RNA alignment metrics for a SAM or BAM file",
+        usage = CollectRnaSeqMetrics.USAGE_SUMMARY + CollectRnaSeqMetrics.USAGE_DETAILS,
+        usageShort = CollectRnaSeqMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRnaSeqMetrics extends SinglePassSamProgram {
+
+    static final String USAGE_SUMMARY = "Produces RNA alignment metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "<p>This tool takes a SAM/BAM file containing aligned RNA transcripts and produces metrics describing the distribution " +
+            "of the bases within the transcripts.  Calculates the total numbers and the percentages of nucleotides within the transcript sub-domains" +
+            " including the untranslated region (UTR), introns, intergenic sequences (between discrete genes) and peptide-coding sequences.  " +
+            "This tool also determines the numbers of bases that pass the sequencing vendor's quality filters (PF_BASES). " +
+            "To learn more about the PF metric, please see the corresponding GATK " +
+            "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6329'><b>dictionary entry</b></a>.</p>" +
+            "" +
+            "<p>Other metrics include the median coverage (depth), 5'/3'-biases, and the numbers of reads with the correct/incorrect " +
+            "strand designation. The 5'/3'-biases represent the bias introduced by reverse transcriptase enzymes during library construction, which" +
+            " result in the over representation of either the 5 or 3' ends of transcripts.  Please see the following " +
+            "<a href=\"https://www.biostars.org/p/102812/#102815\"><b>link</b></a> for information on how this ratio is calculated. </p>" +
+            "" +
+            "<p>The input must be a valid BAM file containing RNAseq data aligned using e.g. <a href=\"http://github.com/alexdobin/STAR\"><b>STAR</b></a>" +
```

Comment:
Good point.  Perhaps the mention of this should go on the command-line-overview page?  

Moreover, I was thinking that perhaps the command-line example should move from Tool Documentation page to the Quick Start page?  

Finally, what about having some links on each of these pages that enables one to easily navigate back and forth among these pages?  For the most part, the links are unidirectional e.g. what about having a block of buttons at the top or bottom of each of these pages that enable the user to go to directly to ToolDocumentation or QuickStart MetricsDefinitions?  I have to say, without some experience in using these, it is difficult for a newbie to find these different pages.